File: Chapter_2/project_1_5.pl
use strict;

printf("Please enter the value of x as a whole number: ");

my $x = <STDIN>;
chomp $x;

my $answer =
  3 * ( $x**5 ) + 2 * ( $x**4 ) - 5 * ( $x**3 ) - ( $x**2 ) + 7 * $x - 6;

print "The answer is $answer \n";

File: Chapter_2/project_1_5.py
from math import pow as p

x = int(input("Enter the value for x: "))
answer = 3 * p(x, 5) + 2 * p(x, 4) - 5 * p(x, 3) - p(x, 2) + 7 * x - 6
print(f"The answer is {answer:.0f}")
